pkg/exchange/bitget/stream.go

```diff
@@ -212,7 +225,19 @@ func parseEvent(in []byte) (interface{}, error) {
 		book.instId = event.Arg.InstId
 		return &book, nil
 
-	case ChannelTrade:
+	case strings.HasPrefix(string(ch), "candle"):
```

Comment:
maybe you can pull out this case to a if statement that runs before this switch, so that you switch cases could be simplified ?

Reply:
Or i pull it out to default? like: 
```

default: 
    if strings.HasPrefix(string(ch), "candle") {
        var kline KLineEvent
		err = json.Unmarshal(event.Data, &kline.Events)
		if err != nil {
			return nil, fmt.Errorf("failed to unmarshal data into KLineEvent, Arg: %+v Data: %s, err: %w", event.Arg, string(event.Data), err)
		}

		kline.actionType = event.Action
                kline.channel = ch
		kline.instId = event.Arg.InstId
		return &kline, nil
    }

    // other case are return err.

```
I hope that every event is handled in switch-case